<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editScientistsRelations</class>
 <widget class="QDialog" name="editScientistsRelations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/Images/DBSimple.jpg</normaloff>:/icons/Images/DBSimple.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(204, 217, 233);
font-family: &quot;Verdana&quot;;
font-size: 9pt;
}

QDialog::dropdown_types{
color: white;
background-color: white;
}

QPushButton{
color: rgb(0, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0.619, x2:1, y2:0.58, stop:0 rgba(176, 176, 176, 255), stop:1 rgba(255, 255, 255, 255));
}

QPushButton::hover{
background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0.318408 rgba(69, 69, 69, 255), stop:1 rgba(162, 162, 162, 255));
}

QPushButton::pressed{
background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0.318408 rgba(69, 69, 69, 255), stop:1 rgba(162, 162, 162, 255));
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget_computers">
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>380</number>
       </attribute>
       <column>
        <property name="text">
         <string>Computers</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;- - - - Computers In Database - - - -&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="dropDown_computers">
       <property name="accessibleName">
        <string/>
       </property>
       <property name="currentText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_add">
         <property name="text">
          <string>Add New Computer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_delete">
         <property name="text">
          <string>Delete </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_done">
         <property name="text">
          <string>Done</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
